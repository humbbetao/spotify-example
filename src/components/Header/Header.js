import React from 'react'

export default function Header({ children = null }) {
  return <header>{children}</header>
}
