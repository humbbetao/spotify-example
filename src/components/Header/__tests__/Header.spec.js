import React from 'react'
import { mount } from 'enzyme'
import Header from '../Header'

describe('Header', () => {
  it('should render correctly when children is not passed', () => {
    const wrapper = mount(<Header />)
    expect(wrapper.children.length).toEqual(1)
    expect(wrapper).toMatchSnapshot()
  })

  it('should render correctly when children is passed', () => {
    const wrapper = mount(
      <Header>
        <p>Teste</p>
      </Header>
    )

    expect(wrapper).toMatchSnapshot()
    expect(wrapper.children.length).toEqual(1)
  })
})
