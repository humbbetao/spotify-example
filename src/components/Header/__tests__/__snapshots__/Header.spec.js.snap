// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly when children is not passed 1`] = `ReactWrapper {}`;

exports[`Header should render correctly when children is passed 1`] = `ReactWrapper {}`;
